front-end/src/helpers/auth.js

```diff
@@ -1,3 +1,16 @@
+import jwt_decode from 'jwt-decode'
+
 export const isLoggedIn = () => {
     return localStorage.getItem('access_token') !== null;
+}
+
+export const getBearerToken = () => {
+    return localStorage.getItem('access_token');
+}
+
+export const getUserIdFromJWT = () => {
+    const token = localStorage.getItem('access_token');
```

Comment:
nit: Since we have `getBearerToken` above which does the same thing, I'd recommend using the function here just for consistency.

Reply:
Updated